middlewares/logger.js

```diff
@@ -0,0 +1,30 @@
+const Student = require('../models/student.model');
+const Tutor = require('../models/tutor.model');
+const jwt = require('jsonwebtoken');
+
+const logger = async (req, res, next) => {
+    try {
+        const { email, password } = req.body;
```

Comment:
This two conditions are almost the same, you can use one try - catch and add conditions in try
calling their respective model, something like this:

![image](https://user-images.githubusercontent.com/36177355/133267679-12f812e1-c51d-4ca7-a283-4dd84d895a4f.png)

Reply:
Hello Leonel, thank you for your review, I made a refactor on the logger